core-backend/wasmtime/src/env.rs

```diff
@@ -40,13 +40,13 @@ use wasmtime::{
 
 /// Data type in wasmtime store
 pub struct StoreData<E: Ext> {
-    pub ext: LaterExt<E>,
+    pub ext: ExtCarrierClone<E>,
 }
```

Comment:
Can be replaced with tuple-struct

Reply:
Better not to do it because you will see `caller.data().0` across the code which is quite ugly

Reply:
> Better not to do it because you will see `caller.data().0` across the code which is quite ugly

agree